package.json

```diff
@@ -2,58 +2,51 @@
   "name": "@playkit-js/playkit-js-providers",
   "version": "2.39.4",
   "description": "",
+  "exports": {
+    "./playkit-ovp-provider": "./dist/playkit-ovp-provider.js",
+    "./playkit-ott-provider": "./dist/playkit-ott-provider.js",
+    "./playkit-analytics-service": "./dist/playkit-analytics-service.js",
+    "./playkit-stats-service": "./dist/playkit-stats-service.js",
+    "./playkit-bookmark-service": "./dist/playkit-bookmark-service.js"
+  },
   "scripts": {
-    "clean": "rm -rf ./dist",
-    "prebuild": "npm run clean",
-    "build": "webpack --mode production",
-    "dev": "webpack-dev-server --mode development",
-    "watch": "webpack --progress --colors --watch --mode development",
+    "build:dev": "webpack serve --open --mode=development",
```

Comment:
upload-to-npm step in Github Actions calls "yarn run build"

see https://github.com/kaltura/ovp-pipelines-pub/blob/e37e02ef9ecb337b28f1142c9ead9f0982804f69/.github/workflows/player_upload_to_npm.yaml#L155